style.css

```diff
@@ -508,3 +530,95 @@ section .languages {
   width: 40px;
   height: 40px;
 }
+
+footer {
+  margin: 60px 0 0 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: url(./Pictures/Final.svg) no-repeat;
+  background-size: 100% auto;
+  width: 100%;
+  height: 779px;
+}
+
+footer form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 60px 0;
+}
+
+.input {
+  width: 327px;
+  height: 48px;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  margin: 8px 0;
+  padding: 0 0 0 16px;
+}
+
+.input-textarea {
+  width: 327px;
+  height: 170px;
+  border: 1px solid #cfd8dc;
+  text-align: start;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  margin: 8px 0;
+  padding: 15px 0 0 16px;
+}
+
+.input::placeholder {
+  font-family: "Poppins", sans-serif;
+  color: #b3bac5;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+}
+
+.input-textarea::placeholder {
+  font-family: "Poppins", sans-serif;
+  color: #b3bac5;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+}
+
+footer .btn {
+  width: 140px;
+  height: 48px;
+  padding: 0.75rem;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  color: #6070ff;
+  background: #fff;
```

Comment:
- [ ] KIndly make sure all the from section has the correct background color. hint `You may adjust the size of your bacground`
![image](https://user-images.githubusercontent.com/97100091/193894003-a6e601fb-4461-4f70-bee0-26c06d14281c.png)

Reply:
The background color of the Contact form has been corrected. Thank you for pointing this out 🤗